org.knime.knip.ilastik/src/org/knime/knip/ilastik/nodes/headless/IlastikHeadlessNodeModel.java

```diff
@@ -381,6 +381,11 @@ private boolean runIlastik(final String tmpDirPath, final List<String> inFiles)
         // build process with project and images
         ProcessBuilder pB = new ProcessBuilder(inFiles);
 
+        // limit cpu + memory usage
+        final Map<String, String> env = pB.environment();
+        env.put("LAZYFLOW_THREADS", String.valueOf(KNIMEConstants.GLOBAL_THREAD_POOL.getMaxThreads()));
```

Comment:
shouldn't this be a property of the node?  I don't see the relation between the KNIME threadpool and Ilastik here.